src/js/engagement_view/src/modules/GraphViz/graph/mergeGraphs.tsx

```diff
@@ -6,15 +6,16 @@ const mergeNodes = (x: Node, y: Node) => {
     let merged = false;
     mapNodeProps(y, (prop: string) => {
         if (!Object.prototype.hasOwnProperty.call(x, prop)) {
-            merged = true;
-            (x as any)[prop] = (y as any)[prop]
+            if ((x as any)[prop] !== (y as any)[prop]) {
```

Comment:
Types?

Reply:
In this case we can't remove the any's, probably ever, due to the nature of this weird reflectiony function.